W-16994661 Limitation note that regex does not match query params (#821)
mulesoft/docs-gateway: mule-gateway/modules/ROOT/pages/policies-resource-level-config-uri-regex.adoc

== Feature Limitations

=== Base Path Requirements

The root path of an API must be well-defined to calculate the base path of the API. Make sure that the path attribute of the HTTP Listener in the Mule application ends with `*`. Otherwise, resource-level policy matching works as if `ignoreBasePath` is set to `false`.

=== Query Parameter Exclusion

> The regex will match only against the URI path component of the request. Query parameters are ignored. For example, a regex pattern like `/api/users.*123` will match requests to `/api/users/123` but won't match a request to `/api/users?id=123`. 

Reviewer: Is it okay to use "will" in this context? If not, I'd recommend saying matches instead of will match in both occurrences in this sentence.

Author: updated to get rid of "will"

> The regex will match only against the URI path component of the request. Query parameters are ignored. For example, a regex pattern like `/api/users.*123` will match requests to `/api/users/123` but won't match a request to `/api/users?id=123`. 

Author: ```suggestion
The regex matches only against the URI path component of the request, and query parameters aren't considered. For example, a regex pattern like `/api/users.*123` matches `/api/users/123` but won't match `/api/users?id=123`. 
```
